server/properties/src/config/log4j.properties

```diff
@@ -50,3 +50,14 @@ log4j.appender.ShineFileAppender.MaxFileSize=10240KB
 log4j.appender.ShineFileAppender.MaxBackupIndex=50
 log4j.appender.ShineFileAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.ShineFileAppender.layout.conversionPattern=%d{ISO8601} %5p [%t] %c{1}:%L - %m%n
+
+# Comment this out to get to performance debug logs.
+# This will generate a lot of logs, so do not turn this on, unless you have been advised to
+#log4j.logger.PerformanceLogger=DEBUG, PerformanceLoggerAppender
```

Comment:
Put this above with the WebRequestsAppender and leave it commented. You can uncomment the below lines so that users don't have to.